public/index.html

```diff
@@ -1,21 +1,44 @@
 <!DOCTYPE html>
 <script src="https://unpkg.com/react@18/umd/react.development.js"></script>
 <script src="https://unpkg.com/react-dom@18/umd/react-dom.development.js"></script>
+<script src="https://unpkg.com/i18next@latest/dist/umd/i18next.js"></script>
+<script src="https://unpkg.com/react-i18next@latest/dist/umd/react-i18next.js"></script>
+<script src="https://unpkg.com/i18next-browser-languagedetector@latest/dist/umd/i18nextBrowserLanguageDetector.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/i18next-http-backend@1.3.1/i18nextHttpBackend.min.js"></script>
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <link rel="stylesheet" href="./index.css">
 <html>
```

Comment:
We’ll need to set the document language to `en-ca` for English and `fr-ca` for French. Since this is a single page app, we will likely need to set a default and then update this with the language switcher.

Reply:
I've always wondered what's the best practice/optimal for SEO: using URL routing to specify the language, or something more dynamic?